.github/workflows/e2e_scripts/cleanup_monitoring.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+from google.cloud import monitoring_v3
+from google.cloud.monitoring_dashboard import v1
+
+project_id = ''
+project_name = 'projects/'
+
+def getProjectId():
+    """Retrieves the project id from the environment variable.
+    Raises:
+        MissingProjectIdError -- When not set.
+    Returns:
+        str -- the project name
+    """
+    project_id = os.environ['GOOGLE_CLOUD_PROJECT']
```

Comment:
This environment variable isn't part of GitHub Actions, so we ideally shouldn't be relying on it here. You should be grabbing the project through the ` ${{ secrets.E2E_PROJECT_ID  }}` injected into the system by GitHub Actions, like the other steps. 

Also, instead of reading from the environment variable, I think it would be more direct to take the project id as an argument and retrieve it with `sys.argv[1]`

Reply:
Gotcha - I'm reading it from the argline and passing in the GitHub Actions secret for the testing script and the gcloud result from the clean.sh script.